docs/shared/connect.md

```diff
@@ -1,5 +1,5 @@
-Confluent Cloud offers pre-built, fully managed connectors that make it easy to instantly connect to popular data sources and end systems in the cloud.
-This recipe shows one example of a data source, but you can substitute your own preferred connector to use any data source.
-The principles are the same, just modify the connector configuration shown below to fit your deployment (see [documentation](https://docs.confluent.io/cloud/current/connectors/index.html)).
+Confluent Cloud offers pre-built, fully managed connectors that make it easy to quickly connect to popular data sources and end systems in the cloud.
+This recipe shows some example data sources, but you can substitute your own connectors to connect to any data supported source.
+The principles are the same, just modify the connector configuration shown below to fit your situation (see [documentation](https://docs.confluent.io/cloud/current/connectors/index.html)).
```

Comment:
```suggestion
The principles are the same, just modify the connector configuration shown below to fit your deployment (see [documentation](https://docs.confluent.io/cloud/current/connectors/index.html)).
```

Reply:
IMHO "deployment" reads funny in this context.  The user's design/architecture/situation is what would determine connector choice, not how they 'deploy' it.